frontend/src/components/shared/FullScreenWrapper/FullScreenWrapper.scss

```diff
@@ -0,0 +1,32 @@
+.streamlit-container .stFullScreenFrame--expanded {
+  position: fixed;
+  top: 0px;
+  left: 0px;
+  bottom: 0px;
+  right: 0px;
+  overflow: auto;
+  background: white;
+  z-index: 1000;
+  padding: 30px;
```

Comment:
Can we make this smaller? Like 0.5rem?
(When I do this I still get a huge margin on the right and bottom sides. No idea why.)